examples/cosmos/ics20/denomTrace.qnt

```diff
@@ -24,17 +24,15 @@ module denomTrace {
   /// as the source zone. When the token is sent back to the chain it previously
   /// received from, the prefix is removed. This is a backwards movement in the
   /// token's timeline and the sender chain is acting as the sink zone."
-  pure def movingBackAlongTrace(hopSource: HopElement, denomTrace: DenomTrace): bool =
+  pure def movingBackAlongTrace(denomTrace: DenomTrace, hopSource: HopElement): bool =
```

Comment:
Could we also mention in the comment above that the `HopElement` passed here describes the source end of the channel?